docs/getting-started/index.md

```diff
@@ -14,15 +20,19 @@ If you are not sure where you should get started, check out our **Guided User St
 
 ## Products
 
-Within the Substrate ecosystem, you can build blockchains, smart contracts, and even user interfaces! Here is a high level overview of the different products which comprise these space:
+Within the Substrate ecosystem, you can build blockchains, smart contracts, and even user interfaces! Here is a quick overview of the different products which comprise these spaces:
 
-* [Substrate](https://github.com/paritytech/substrate): A next generation blockchain development framework built using Rust and Wasm.
-* [ink!](https://github.com/paritytech/ink): A Rust based eDSL for building Wasm smart contracts compatible with Substrate's Contract module.
+* [Substrate](https://github.com/paritytech/substrate): A next generation blockchain development framework built using WebAssembly, Libp2p and GRANDPA Consensus.
```

Comment:
```suggestion
* [Substrate](https://github.com/paritytech/substrate): A next generation blockchain development framework built using WebAssembly, Libp2p, and GRANDPA Consensus.
```

Why is GRANDPA consensus called out here? Where did Rust go?
Again, I think we need direction from our business/marketing for these kinds of statements.

Reply:
This description is from the [README](https://github.com/paritytech/substrate#2-description), I was thought to make it consistent.